pkg/urbit/configure

```diff
@@ -17,6 +17,14 @@ defmacro () {
 
 defmacro URBIT_VERSION "\"$URBIT_VERSION\""
 
+# propogate this info into config.h, so that we can know what __attribute__ style to use in aliases.
+if [ ${CC-cc} == "gcc" ]
+then
+     defmacro GCC 1
+else
+     defmacro GCC 0
+fi	
```

Comment:
This macro is no longer necessary, please remove.

Reply:
done